src/common/components/paymentMethods/editPaymentMethod/editPaymentMethod.modal.tpl.html

```diff
@@ -0,0 +1,72 @@
+<div class="account-management">
+  <div class="modal-header">
+    <div class="container-fluid">
+      <div class="row">
+        <div class="col-xs-12">
+          <div class="border-bottom-small">
+            <button type="button"
+                    class="close"
+                    aria-label="Close"
+                    ng-click="$ctrl.dismiss()">
+              <span aria-hidden="true">×</span>
+            </button>
+            <h3 class="text-center">Edit Payment Method</h3>
+          </div>
+        </div>
+      </div>
+    </div>
+  </div>
+
+  <div class="modal-body pt0">
+    <div class="container-fluid" tabindex="-1" role="dialog">
+      <div class="row">
+        <div class="col-md-12 col-xs-12">
+          <div class="alert alert-danger" role="alert" ng-if="$ctrl.resolve.submissionError.error">
+            <p ng-if="$ctrl.resolve.submissionError.error" ng-switch="$ctrl.resolve.submissionError.error">
```

Comment:
I think we should make a component for this or include it in an existing one. I've copied and pasted it a couple of places already. You can leave it for now. I can make a PR for it or you can if you get to it. Maybe it should go in https://github.com/CruGlobal/give-web/blob/master/src/common/components/paymentMethods/addNewPaymentMethod/addNewPaymentMethod.tpl.html. I guess earlier I was hesitant to have to pass the errors further down the component tree.